include/fi_file.h

```diff
@@ -46,7 +47,6 @@ extern "C" {
 
 int fi_read_file(const char *dir, const char *file, char *buf, size_t size);
 int fi_poll_fd(int fd, int timeout);
-int fi_fd_nonblock(int fd);
```

Comment:
on windows platform it is static inline, on unix there is more complex implementation